src/components/Grid/useGridSorting.module.scss

```diff
@@ -1,6 +1,6 @@
 /**
  * Sort icons UX (from default, up, to down) seem to follow the same
- * conventions as Bootsprap: https://examples.bootstrap-table.com/
+ * conventions as Bootstrap: https:\/\/examples.bootstrap-table.com/
```

Comment:
This gets around a script we have that checks for improper use of `//` in `.module.scss` files